src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -75,7 +76,7 @@ the MIT License. See LICENSE in the project root for license information. -->
                         </StackPanel>
                         <StackPanel Grid.Column="0" Grid.Row="1" Margin="0,0,100,0">
                             <Controls:NumberBox x:Uid="Profile_FontSize" Margin="0,0,0,20" FontSize="15" Value="{x:Bind Profile.FontSize, Mode=TwoWay}" SpinButtonPlacementMode="Compact" SmallChange="1" LargeChange="10" ToolTipService.Placement="Mouse" />
-                            <TextBox x:Uid="Profile_FontWeight" Margin="0,0,0,20" FontSize="15" ToolTipService.Placement="Mouse"/>
+                            <TextBox x:Uid="Profile_FontWeight" Margin="0,0,0,20" FontSize="15" Text="{x:Bind Profile.FontWeight, Converter={StaticResource FontWeightConverter}, Mode=TwoWay}" ToolTipService.Placement="Mouse"/>

```

Comment:
feel like this should be a dropdown or something; leon's work in #8086 may help?